.dev/tests/cypress/support/commands.js

```diff
@@ -1,14 +1,11 @@
-import { disableGutenbergFeatures, isNotWPLocalEnv, loginToSite } from '../helpers';
+import { disableGutenbergFeatures, goTo, loginToSite } from '../helpers';
 
 before( function() {
 	loginToSite().then( () => {
-		if ( isNotWPLocalEnv() ) {
-			// Waiting to see if the Welcome Guide will show up. Could probably be improved, but
-			// for the moment, it seems hard to tie the wait to something else
-			cy.wait( 10000 );
-		}
-
-		disableGutenbergFeatures();
+		goTo( '/wp-admin/post-new.php?post_type=post' ).then( () => {
+			cy.wait( 2000 );
```

Comment:
Is there a reason why you still execute it in local now? In my tests, the wait was not needed, so that is a 2 seconds which is useless in that case.

Reply:
Yeah, I could not get this working in GitHub Actions consistently with Cypress. Seems the code executes too fast, even though it is waiting on the data object. We would possibly add some additional check for the featureActive portion, but will suffice until we can complete the Playwright migration.

Reply:
Yeah, no problem on the waiting on the CI (we dit 10s before, so 2s is an improvement). I was more wondering why you removed the if that made the tests not wait in local (as it is not needed).